lib/remote_coffee_slack/session_direct_message_notifier.rb

```diff
@@ -0,0 +1,39 @@
+module RemoteCoffeeSlack
+  class SessionDirectMessageNotifier
+    def initialize(client, coffee_mates)
+      @client = client
+      @coffee_mates = coffee_mates
+    end
+
+    def perform
+      conversation_channels.each do |channel_id|
+        client.chat_postMessage(
+          channel: channel_id,
+          text: next_session_message,
+          as_user: true)
+      end
+
+      true
```

Comment:
Just curious, do you think would it ever be a scenario where it returns `false`?

Reply:
I don't have in mind a falsy scenario, I think we can start in this way and iterate :)